aries_cloudagent/config/argparse.py

```diff
@@ -1245,6 +1248,8 @@ def get_settings(self, args: Namespace) -> dict:
 
         if args.emit_did_peer_2:
             settings["emit_did_peer_2"] = True
+        if args.emit_did_peer_4:
+            settings["emit_did_peer_4"] = True
```

Comment:
Wondering when one of these will become the new default, rather than having to specify the behaviour at startup (I am a bit out of the loop on this so I am likely missing some background info).